src/server/replica.cc

```diff
@@ -809,18 +809,18 @@ void DflyShardReplica::StableSyncDflyReadFb(Context* cntx) {
       break;
 
     last_io_time_ = Proactor()->GetMonotonicTimeNs();
-
-    if (!tx_data->is_ping) {
+    if (tx_data->opcode == journal::Op::PING) {
```

Comment:
it's just for counting stats ?

Reply:
no we send ack from replica to master when we get ping